### DISCUSSION TITLE
Disconnect from the server

### DISCUSSION THREAD

**Discussion by choonk** [PrismarineJS/mineflayer#2309] [Q&A]
Does anyone know how to disconnect from the server? It's not working for me and I'm getting the error "Cannot read propetty of 'quit' of undefined". I've also tried bot.end() as the docs described, but both return this error. Any ideas? Thank you.

`if (msg.startsWith("Woah there, slow down")) {
       bot.quit("Relogging")`

**Comment by U5B** [comment]
Could you paste all of your code?

That error means 'bot' is not defined.
